daml-lf/interpreter/perf/src/main/scala/com/daml/lf/explore/Explore.scala

```diff
@@ -62,7 +65,7 @@ object PlaySpeedy {
   }
 
   private val noPackages =
-    data.assertRight(PureCompiledPackages(Map.empty, Compiler.NoStackTrace, Compiler.NoProfile))
+    PureCompiledPackages(Map.empty, Map.empty, compilerConfig)
```

Comment:
afaict, we had disabled stack tracing here before now it’s enabled. I guess it doesn’t make a difference?